src/cmd/apply.go

```diff
@@ -123,6 +132,11 @@ func UpdateTheme() {
 	updateCSS()
 	utils.PrintSuccess("Custom CSS is updated")
 
+	if injectJS {
+		pushThemeExtension()
+		utils.PrintSuccess("Custom JS is updated")
```

Comment:
`Custom JS` but `dedicated theme script` *?* Decide on description what it's supposed to do. Either choose that it's javascript related to themes or custom js file that has nothing to do with themes